openssl-dynamic/src/main/java/io/netty/internal/tcnative/SSLPrivateKeyMethodTask.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.internal.tcnative;
+
+abstract class SSLPrivateKeyMethodTask extends SSLTask {
+    private static final byte[] EMPTY = new byte[0];
```

Comment:
Didn't we have `EmptyArrays` in `netty-common`? Please feel free to ignore this if `netty-tcnative` does not have a dependency on `netty-common`.

Reply:
it has not :)